waspc/examples/todoApp/ext/actions.js

```diff
@@ -25,6 +27,7 @@ export const updateTaskIsDone = async ({ id, isDone }, context) => {
     throw new HttpError(403)
   }
 
+  await sleep(3000);
```

Comment:
Added these sleeps in for demonstration purposes. Will remove if we ever merge this.